src/test/java/step3/domain/LottoResultTest.java

```diff
@@ -0,0 +1,33 @@
+package step3.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class LottoResultTest {
+    private LottoResult lottoResult;
+
+    @BeforeEach
+    void beforeEach() {
+        lottoResult = new LottoResult();
+    }
+
+    @Test
+    void 순위별_당첨횟수_확인() {
+        assertThat(lottoResult.rankingCount(Ranking.FIRST)).isEqualTo(0);
```

Comment:
isZero() 를 사용해 볼 수도 있을 것 같네요 😄 
```suggestion
    @Test
    void 순위별_당첨횟수_확인() {
        assertThat(lottoResult.rankingCount(Ranking.FIRST)).isZero();
```

Reply:
isZero를 계속 놓치네요 😢 
감사합니다!